### DISCUSSION TITLE
Does the Beeper Mini app always have to be connected?

### DISCUSSION THREAD

**Discussion by pcpumpkin** [thatmarcel/beepserv-rewrite#18] [Q&A]
Hi, I am using combination of Beeper Mini and Bluebubbles.  I do use Bluebubbles app on my Android device over Beeper Mini more. I seem to get periods when my phone number is deregistered frequently and there is a lag when it gets re-registered.  Is this because Beeper Mini is closing in the background from lack of use causing the delay?  I have not upgraded to the new 78d56e4 version yet.  Will the newest version improve the periods of phone number deregistration?

**Comment by thatmarcel** [comment]
The newer versions should not improve the de-registration frequency. However, turning iMessage off on the iPhone _might_ help.

If you're using the newest version of the [Beeper app](https://www.beeper.com/download), you should be able to manually re-register in the app settings.
